.vscode/settings.json

```diff
@@ -23,5 +23,5 @@
   },
   "javascript.preferences.importModuleSpecifier": "relative",
   "typescript.preferences.importModuleSpecifier": "relative",
-  "prettier.configPath": ".prettierrc.js",
-} 
+  "prettier.configPath": ".prettierrc.js"
+}
```

Comment:
Exclude `settings.json` from this PR.

Reply:
I restored this file through  [this way](https://stackoverflow.com/questions/2364147/how-to-get-just-one-file-from-another-branch).

Reply:
`https://stackoverflow.com/questions/2364147/how-to-get-just-one-file-from-another-branch`